helix-term/src/commands/typed.rs

```diff
@@ -656,7 +656,7 @@ pub fn write_all_impl(
     force: bool,
     write_scratch: bool,
 ) -> anyhow::Result<()> {
-    let mut errors: Vec<&'static str> = Vec::new();
+    let mut errors: Vec<String> = Vec::new();
```

Comment:
I kept this as String since it will allow changes to the error messages easier and there is low overhead here